apps/experiments/views/experiment.py

```diff
@@ -485,9 +491,14 @@ def get_initial(self):
     def get_object(self, queryset=None):
         obj = super().get_object(queryset)
         if obj.working_version:
-            raise Http404("Cannot edit experiment versions.")
+            raise Http404("Experiment not found.")
         return obj
 
+    def post(self, request, *args, **kwargs):
+        if self.get_object().is_archived:
+            raise PermissionDenied("Cannot edit archived experiments.")
```

Comment:
👍🏻

Reply:
Should this be applied to other places too like 'create version' etc.?

Reply:
Sure yeah. I double checked that one cannot accept an invite to chat to an experiment or use the public link once it is archived. It will return a 404